src/components/BoardWizard.vue

```diff
@@ -0,0 +1,333 @@
+                :value="list.id"
+                v-model="productionList"
+              >
+              <label :for="`prod_${list.id}`" class="checkbox" v-on:click="radioListChanged"></label>
+              <label :for="`prod_${list.id}`" class="wizard--text-list" v-on:click="radioListChanged">
+                {{ list.name }}
+              </label>
+            </div>
+          </div>
+        </div>
+        <div>
+          <p class="wizard--text wizard--text-secondary">Archived</p>
+          <div class="wizard--container wizard--container-list wizard--container-list-secondary">
+            <div
+              class="checkbox-container checkbox-container-secondary"
+              v-for="list in selectedListsObjects"
+              :key="list.id"
+            >
+              <input
+                type="checkbox"
+                :id="`arc_${list.id}`"
+                style="display: none;"
+                :value="list.id"
+                v-model="archivedLists"
+              >
+              <label :for="`arc_${list.id}`" class="checkbox" v-on:click="generalListChanged"></label>
+              <label :for="`arc_${list.id}`" class="wizard--text-list" v-on:click="generalListChanged">
+                {{ list.name }}
+              </label>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="wizard--button-container">
+        <button id="back_button" class="button button-disabled" v-on:click="stepBack" disabled>BACK</button>
+        <button id="save_button" class="button button-save button-disabled" v-on:click="saveData" disabled>NEXT</button>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import BoardBox from './BoardBox.vue';
+import { get, save } from '../utils/configurationPersistance.js';
+import { request, onRequestError } from '../utils/trelloManager.js';
+
+const secondStageNumber = 2;
+
+export default {
+  name: 'BoardWizzard',
+  props: {
+    boards: Array,
+  },
+  components: {
+    BoardBox,
+  },
+  data() {
+    return {
+      selectedBoard: null,
+      stage: 0,
+      allLists: [],
+      selectedLists: [],
+      archivedLists: [],
+      endList: null,
+      wipLists: [],
+      backlogList: null,
+      productionList: null,
+      toLoad: true,
+    };
+  },
+  computed: {
+    selectedBoardObject() {
+      return this.boards.find((board) => board.id === this.selectedBoard);
+    },
+    selectedListsObjects() {
+      return this.allLists.filter((list) => this.selectedLists.includes(list.id));
+    },
+  },
+  mounted() {
+    this.selectedBoard = get('boards', null);
+    if (this.selectedBoard !== null) {
+      this.selectedBoard = this.selectedBoard[0]
+      document.getElementById(this.selectedBoard)
+        .classList.toggle('wizard--board-selected');
+      document.getElementById('save_button').classList.remove('button-disabled');
+      document.getElementById('save_button').disabled = false;
+    }
+  },
+  updated() {
```

Comment:
Method 'updated' has too many statements (14). Maximum allowed is 10   max-statements